demo/sizzle.sysl

```diff
@@ -0,0 +1,318 @@
+# This project tells our integration diagram what apps to include
+import mastercard.yaml as MasterCard
+
+# This app specifies the projects to include, it is optional and can be left out
+Sizzle[~project]:
+    @contact.name = "Jimmy Smith"
+    @description =:
+        | Sizzle is a Gourmet Sausage Restaurant.
+        |
+        | We aim to provide the truly Aussie sausage sizzle 
+        | to our customers.
+        |
+        | We will offer 90 minute table allocations and 
+        | comply with all directions and safety procedures 
+        | implemented by the state government.
+        |
+        | You can dine-in, take away, or shop online.
+    @version = "1.0.0"
+
+    Frontend:
+        Application
+    Backend:
+        BFF
+        Database
+
+
+Dine-in Customer [~human]:
+    # ~human specifies a user
+    Menu:
+        Mobile <- Menu
+    Order:
+        Mobile <- Order
+    PlaceOrder:
+        Mobile <- PlaceOrder
+    Pay:
+        Dashboard <- Pay
+    Review:
+        Mobile <- Review
+
+Online Customer [~human]:
+    Sign up:
+        Website <- Signup
+    Login:
+        Website <- Login
+        Website <- Profile
+    Change password:
+        Website <- ChangePassword
+    Menu:
+        Website <- Menu
+    Order:
+        Website <- Order
+    Place and Pay Order:
+        Website <- PlaceOrder
+        Website <- Pay
+    Review:
+        Website <- Review
+
+
+Mobile [~ui]:
```

Comment:
Maybe we can duplicate this so that we have both iOSApp and AndroidApp?

Reply:
I wanna keep this simple at first. And in an ideal world, the iOSApp and AndroidApp are supposed to provide the same functionality.